transport_nantes/topicblog/views.py

```diff
@@ -728,6 +728,34 @@ def get_unsubscribe_link(self) -> str:
         """
         pass
 
+
+class UnsubscribeFromMailingListView(TemplateView):
+    """
+    View to unsubscribe a user from a mailing list.
+    """
```

Comment:
In view of the name of the view, this comment doesn't appear to be adding much value. ;-)